doc/manual/profile.rst

```diff
@@ -211,7 +211,7 @@ Options for controlling the display of profile results
 :func:`Profile.print` has more options than we've described so far.
 Let's see the full declaration::
 
-    function print(io::IO = STDOUT, data = fetch(); format = :tree, C = false, combine = true, cols = tty_cols())
+    function print(io::IO = STDOUT, data = fetch(); format = :tree, C = false, combine = true, cols = tty_cols(), maxdepth = typemax(Int), sortedby = :filefuncline)
```

Comment:
I don't know what you mean by settles out. It's in an unfortunate state right now and other than better clarifications of how the current state is supposed to work I don't think it's going to change dramatically before 0.4, we just have to cope as best we can. Or decide to revert the conversion again since it's clearly confusing everyone.